### PR TITLE
Market Functions: LimitOrders & OrderBook

### PR DESCRIPTION
**Pull request by technologiespro**
```
// get order book max depth 50
let ob = await BitShares.getOrderBook("BTS","XBTSX.STH", 10);
console.log(ob);
```

```
// get limit orders
let ob = await BitShares.getLimitOrders("BTS","XBTSX.STH", 10);
console.log(ob);
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.7.0",
+    "nconf": "^0.10.0",
```
Comment (scientistnik) [inline package.json]: What is it?

File: lib/bitshares.js [hunk 2]
```diff
@@ -159,6 +159,24 @@ export default class BitShares {
     )
   }
 
+    static async getLimitOrders(quoteSymbol, baseSymbol, limit = 50) {
+    if (limit > 100) limit = 100;
```
Comment (scientistnik) [inline lib/bitshares.js]: please use 2 space for indent

File: lib/bitshares.js [hunk 3]
```diff
@@ -159,6 +159,24 @@ export default class BitShares {
...
+        return BitShares.db.get_limit_orders(
```
Comment (scientistnik) [inline lib/bitshares.js]: This static function, use `this` without `BitShares`.
Generally, I want to rewrite this (and other) class for use arrow functions in the future.

File: package.json (hunk 1 above)
Comment (technologiespro) [reply package.json]: this for test config

File: lib/bitshares.js (hunk 3 above)
Comment (technologiespro) [reply lib/bitshares.js]: ok, thx

**Pull request merged**